kubernetes/kubernetes.go

```diff
@@ -45,49 +46,52 @@ type ClusterVersion struct {
 
 //KubeAgentConfig K8s agent configuration
 type KubeAgentConfig struct {
-	APIKey                string
-	BearerToken           string
-	BearerTokenPath       string
-	Cert                  string
-	ClusterName           string
-	ClusterHostURL        string
-	clusterUID            string
-	HeapsterOverrideURL   string
-	HeapsterURL           string
-	Key                   string
-	OutboundProxyAuth     string
-	OutboundProxy         string
-	provisioningID        string
-	RetrieveNodeSummaries bool
-	GetAllConStats        bool
-	ForceKubeProxy        bool
-	Insecure              bool
-	OutboundProxyInsecure bool
-	UseInClusterConfig    bool
-	CollectHeapsterExport bool
-	PollInterval          int
-	ConcurrentPollers     int
-	CollectionRetryLimit  uint
-	failedNodeList        map[string]error
-	AgentStartTime        time.Time
-	Clientset             kubernetes.Interface
-	ClusterVersion        ClusterVersion
-	HeapsterProxyURL      url.URL
-	OutboundProxyURL      url.URL
-	HTTPClient            http.Client
-	NodeClient            raw.Client
-	InClusterClient       raw.Client
-	msExportDirectory     *os.File
-	TLSClientConfig       rest.TLSClientConfig
-	Namespace             string
-	ScratchDir            string
-	NodeMetrics           EndpointMask
-	ParseMetricData       bool
+	APIKey                 string
+	BearerToken            string
+	BearerTokenPath        string
+	Cert                   string
+	ClusterName            string
+	ClusterHostURL         string
+	clusterUID             string
+	HeapsterOverrideURL    string
+	HeapsterURL            string
+	Key                    string
+	OutboundProxyAuth      string
+	OutboundProxy          string
+	provisioningID         string
+	RetrieveNodeSummaries  bool
+	GetAllConStats         bool
+	ForceKubeProxy         bool
+	Insecure               bool
+	OutboundProxyInsecure  bool
+	UseInClusterConfig     bool
+	CollectHeapsterExport  bool
+	PollInterval           int
+	CollectionRetryLimit   uint
+	failedNodeList         map[string]error
+	AgentStartTime         time.Time
+	Clientset              kubernetes.Interface
+	ClusterVersion         ClusterVersion
+	HeapsterProxyURL       url.URL
+	OutboundProxyURL       url.URL
+	HTTPClient             http.Client
+	NodeClient             raw.Client
+	InClusterClient        raw.Client
+	msExportDirectory      *os.File
+	TLSClientConfig        rest.TLSClientConfig
+	Namespace              string
+	ScratchDir             string
+	NodeMetrics            EndpointMask
+	Informers              map[string]*cache.SharedIndexInformer
+	InformerResyncInterval int
+	ConcurrentPollers      int
+	ParseMetricData        bool
```

Comment:
Was the functionality for this flag re-implemented for informers?

Reply:
no, going back and implementing these changes

Reply:
With most recent changes, yes this functionality got re-implemented :)